src/modules/notificationAlert/i18n/es_ES.root.json

```diff
@@ -17,7 +17,7 @@
 		"dm": "Nuevo mensaje en la bandeja de entrada",
 		"ingame_news": "Noticias en el juego",
 		"mission_getred": "Una misión requiere atención",
-		"mission_getred_alliance": "Una misión de alianza requiere atención
```

Comment:
@Suchty112  a "at the end before the ,

Reply:
yes I change immediately with the last error of jan. thank you